libnmstate/nm/plugin.py

```diff
@@ -120,6 +123,21 @@ def get_interfaces(self):
 
             info.append(iface_info)
 
+        connections = self.client.get_connections()
+        for con in connections:
+            iface_name = con.get_interface_name()
+            if con.get_visible() and iface_name not in current_ifaces:
+                con_info = {
+                    Interface.NAME: iface_name,
+                    Interface.TYPE: nm_translator.Nm2Api.get_iface_type(
+                        con.get_connection_type()
+                    ),
+                    Interface.STATE: InterfaceState.DOWN,
+                    Interface.IPV4: {InterfaceIP.ENABLED: False},
```

Comment:
Showing wrong information is worse than no data.

Please remove these IPv4 and IPv6 info.

Reply:
Done, thanks!